sql/moz-fx-data-shared-prod/telemetry_derived/unified_metrics_v1/checks.sql

```diff
@@ -11,9 +11,8 @@
 {{ value_length(column="client_id", expected_length=36, where="submission_date = @submission_date") }}
 
 #warn
-{{ is_unique(columns=["client_id"], where="submission_date = @submission_date") }}
```

Comment:
This table should have a grain of 1 entry per client per partition. If this checks fails then there's something going wrong, by just deleting the check we're ignoring the source issue.

Reply:
If that's the case, this should be a `fail` warning and someone should investigate. I do not think we should go into the freeze with a check that's known to fail (and was known to fail historically)

Reply:
I think the reason we decided to go with `warn` now is to avoid breaking jobs and progressively switch over to `fail` mode for some checks once we get a feel for how many things fail and do not adhere to our assumptions about the data.

But I agree, this should become a fail check. However, before we do that switch here we should address the duplication inside of the Fenix tables. I'll push a commit to this branch commenting this check out along with a reference to Fenix duplication bug explaining why for now we chose to omit it.

Reply:
That's perfect, it's all merged. Thanks for rounding this out!